tests/Platforms/AbstractPlatformTestCase.php

```diff
@@ -398,16 +391,16 @@ public function testQuotedColumnInForeignKeyPropagation(): void
             'FK_WITH_RESERVED_KEYWORD',
         );
 
-        // Foreign table with non-reserved keyword as name (does not need quotation).
+        // Foreign table with non-reserved keyword as name
         $foreignTable = new Table('foo');
 
-        // Foreign column with reserved keyword as name (needs quotation).
+        // Foreign column with reserved keyword as name
         $foreignTable->addColumn('create', Types::STRING);
```

Comment:
Technically, this part of the test is redundant since all identifiers are now rendered in SQL consistently regardless of whether they are keywords or not. But redesigning the test to correspond to the new design is a bit too much work.